package.json

```diff
@@ -24,19 +26,25 @@
     "node": ">=4.0.0"
   },
   "dependencies": {
+    "@mapbox/polyline": "^0.2.0",
+    "async": "^2.5.0",
     "cli-table2": "^0.2.0",
     "csv-parse": "^1.2.0",
     "express": "^4.14.0",
+    "fs-extra": "^3.0.1",
+    "jsftp": "^2.0.0",
     "node-postal": "^1.0.0",
     "pbf2json": "^4.2.0",
-    "@mapbox/polyline": "^0.2.0",
+    "pelias-config": "^2.11.0",
```

Comment:
I believe that's already the case. All should be downloaded if no config is found.